Svc/FpySequencer/FpySequencerDirectives.cpp

```diff
@@ -289,7 +289,7 @@ Signal FpySequencer::getPrm_directiveHandler(const FpySequencer_GetPrmDirective&
 
 Signal FpySequencer::cmd_directiveHandler(const FpySequencer_CmdDirective& directive, DirectiveError& error) {
     Fw::ComBuffer cmdBuf;
-    Fw::SerializeStatus stat = cmdBuf.serialize(Fw::ComPacketType::FW_PACKET_COMMAND);
+    Fw::SerializeStatus stat = cmdBuf.serialize(static_cast<FwPacketDescriptorType>(Fw::ComPacketType::FW_PACKET_COMMAND));
```

Comment:
Could use the new form serializeFrom() variant. serialize() will be deprecated next release.

Reply:
I agree. I'll open a ticket asking for this change. We can address it in a separate PR

Reply:
Here is the ticket https://github.com/nasa/fprime/issues/4011